app/posts/find-teacher-training/2024-06-18-improving-course-pages.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Improving how candidates view course information and compare courses
+description: Reducing the length of course pages and moving information to be accessible at the point of need.
+date: 2024-06-18
+tags:
+  - courses
+  - locations
+  - placement schools
+  - search results
+  - entry requirements
+  - international candidates
+  - visa sponsorship
+  - accredited providers
+screenshots:
+  items:
+    - text: Course page prototype - collapsed
+      src: course-page-protoype-collapsed.png
+    - text: Course page prototype - expanded
+      src: course-page-prototype-expanded.png
+---
+
+## The issue
+
+### Previous user research
+
+Previous research conducted in February 2022 found that:
+
+- candidates find the course pages are long and feel overwhelming​.
+- course information is often very similar, making it hard for candidates to know which are right for them​
+
+Some commented that they would prefer for the pages to contain only key points, and to be formatted in a way that allows for quick scans, for example with bullet points.
+
+> "It's quite tiring having to read a lot after work / day job, I want something that supports me in doing a quick information scan" - Career Changer
+
+It’s not clear what is the role of Find vs providers’ websites​​, and how they should be using each; and for some candidates it becomes a game of ‘spot the difference’.​
+
+> "I wouldn't use the Gov website as the 'sole course' for researching and getting information. I would likely look at the provider’s website instead." - Career Changer​
+
+Some proposed an accordion structure so that they’re less overwhelmed by the sheer amount of text.​
+
+Most course pages look fairly identical, so candidates look for small differentiating factors that might win them over.​
+
+> "It's all very samey so it’s hard to tell the difference between courses" - Recent Graduate
+
+Information about the course can often be long-winded and ‘meaningless’, with quality varying massively between providers.​
```

Comment:
This feels a bit punchy for a public blog post that providers could see? Could we say that Course descriptions can be longer than necessary and the usefulness of the information contained in them varies. ?

Reply:
Good spot, I took this straight from a previous deck but it wouldn't have that audience in mind.